VS-Code.md

```diff
@@ -0,0 +1,53 @@
+# 5 feature of VS- Code 
```

Comment:
Some changes:

- Please try to add bullet points.
- If the heading says "5 points" please don't add more than 5 points for the heading. Please remove extra points.